### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by selvach**
This PR upgrades two dependencies that have had security advisory
- Upgrade react-native-webview
- Upgrade axios

### CODE REVIEW

**Review by bionara — COMMENTED** [review]
Code looks fine. Will pull and run tomorrow morning and see if any issues.

File: yarn.lock [hunk 1]
```diff
@@ -4072,12 +4072,12 @@ axios-mock-adapter@1.19.0:
...
-  version "0.19.2"
-  resolved "https://registry.yarnpkg.com/axios/-/axios-0.19.2.tgz#3ea36c5d8818d0d5f8a8a97a6d36b86cdc00cb27"
-  integrity sha512-fjgm5MvRHLhx+osE2xoekY70AhARk3a6hkN+3Io1jc00jtquGvxYlKlsFUhmUET0V5te6CcZI7lcv2Ym61mjHA==
+axios@0.21.1:
+  version "0.21.1"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.21.1.tgz#22563481962f4d6bde9a76d516ef0e5d3c09b2b8"
+  integrity sha512-dKQiRHxGD9PPRIUNIWvZhPTPpl1rf/OxTYKsqKUDjBwYylTvV7SjSHJb9ratfyzM6wCdLCOYLzs73qpg5c4iGA==
   dependencies:
-    follow-redirects "1.5.10"
+    follow-redirects "^1.10.0"
```
Comment (petermason) [inline yarn.lock:4080]: remove ^

Comment (selvach) [reply yarn.lock:4080]: All of our top level packaages don't have the "caret" - but the yarn.lock files that are generated which have depdencies themselves all appear to have the "caret". Do you suggestion changing all of them?

Comment (petermason) [reply yarn.lock:4080]: hmm, good point, probably okay to ignore this but worth noting that this is happening.

**Review by petermason — APPROVED** [review]

**Review by bionara — APPROVED** [review]
Checked out - compiles and works as expected. Approved!

**Pull request merged**